### PR TITLE
[IOTDB-1180]Optimize log file name

### PR DESCRIPTION
**Pull request by yanhongwangg**
https://issues.apache.org/jira/browse/IOTDB-1180

### CODE REVIEW

**Comment by HTHou** [comment]
Hi, I think it would be better to state the meaning of the numbers in the new log file name clearly...

**Comment by yanhongwangg** [comment]
> Hi, I think it would be better to state the meaning of the numbers in the new log file name clearly...

OK, I added details in JIRA

File: server/src/assembly/resources/conf/logback.xml [hunk 1]
```diff
@@ -25,14 +25,12 @@
     <statusListener class="ch.qos.logback.core.status.NopStatusListener"/>
     <appender class="ch.qos.logback.core.rolling.RollingFileAppender" name="FILEERROR">
         <file>${IOTDB_HOME}/logs/log_error.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${IOTDB_HOME}/logs/log-error.%i.log.gz</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>5</maxIndex>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
```
Comment (mychaow) [inline server/src/assembly/resources/conf/logback.xml:28]: so how to limit the log size?

Comment (jixuan1989) [reply server/src/assembly/resources/conf/logback.xml:28]: is there any upper bound for each file? and the total size of files?

Comment (yanhongwangg) [reply server/src/assembly/resources/conf/logback.xml:28]: OK, I'll add restrictions

Comment (yanhongwangg) [reply server/src/assembly/resources/conf/logback.xml:28]: > so how to limit the log size?

I'll add restrictions

Comment (yanhongwangg) [reply server/src/assembly/resources/conf/logback.xml:28]: > is there any upper bound for each file? and the total size of files?

Copy that, I'll add it

File: server/src/assembly/resources/conf/logback.xml [hunk 2]
```diff
@@ -109,14 +101,12 @@
...
     <appender class="ch.qos.logback.core.rolling.RollingFileAppender" name="FILETRACE">
         <file>${IOTDB_HOME}/logs/log_trace.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${IOTDB_HOME}/logs/log-trace.%i.log.gz</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>5</maxIndex>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>${IOTDB_HOME}/logs/log-trace-%d{yyyyMMdd-HHmmss}.%i.log.gz</fileNamePattern>
+            <timeBasedFileNamingAndTriggeringPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP">
+                <maxFileSize>10MB</maxFileSize>
```
Comment (jixuan1989) [inline server/src/assembly/resources/conf/logback.xml:107]: 50MB

File: server/src/assembly/resources/conf/logback.xml [hunk 3]
```diff
@@ -88,14 +82,12 @@
...
     <appender class="ch.qos.logback.core.rolling.RollingFileAppender" name="FILEDEBUG">
         <file>${IOTDB_HOME}/logs/log_debug.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${IOTDB_HOME}/logs/log-debug.%i.log.gz</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>5</maxIndex>
+        <rollingPolicy class="ch.qos.logback.core.rolling.TimeBasedRollingPolicy">
+            <fileNamePattern>${IOTDB_HOME}/logs/log-debug-%d{yyyyMMdd-HHmmss}.%i.log.gz</fileNamePattern>
+            <timeBasedFileNamingAndTriggeringPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP">
+                <maxFileSize>10MB</maxFileSize>
```
Comment (jixuan1989) [inline server/src/assembly/resources/conf/logback.xml:88]: 50MB

**Review by jixuan1989 — APPROVED** [review]

**Pull request merged**